src/main/java/dinkplugin/DinkPlugin.java

```diff
@@ -240,6 +241,11 @@ public void onChatMessage(ChatMessage message) {
         chatNotifier.onMessage(message.getType(), source, chatMessage);
         switch (message.getType()) {
             case GAMEMESSAGE:
+                if (Objects.equals("runelite", message.getSender())) {
```

Comment:
```suggestion
                if ("runelite".equals(source)) {
```

I think in https://redirect.github.com/lalochazia/missed-clues/pull/2 the `name` was populated, not `sender`: https://github.com/runelite/runelite/blob/56634bac9e1e9263e0e18bbab3caaf70d8ff3da6/runelite-api/src/main/java/net/runelite/api/Client.java#L116

also yoda-style ordering is already null safe, so can optimize a lil

Reply:
you are right, brain fart